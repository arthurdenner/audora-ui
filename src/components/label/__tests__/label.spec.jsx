import React from 'react';
import { render } from 'enzyme';
import Label from '../label';

describe('Label', () => {
  it('renders correctly', () => {
    const wrapper = render(<Label content="Test Label" />);
    expect(wrapper).toMatchSnapshot();
  });
});
