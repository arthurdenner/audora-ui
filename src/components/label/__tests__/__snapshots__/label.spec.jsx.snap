// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
<label
  class="sc-bdVaJa xPDeO"
>
  Test Label
</label>
`;
